.travis.yml

```diff
@@ -10,4 +10,4 @@ script:
   - GO111MODULE=on go test -v -race ./...
   - git diff --exit-code
 install:
-  - GO111MODULE=off go get golang.org/x/lint/golint
+  - GO111MODULE=off go get golang.org/x/lint/golin
```

Comment:
Is this change required?

Reply:
Repo is ok .. my  workspace has the typo.. checking why

Reply:
This is still broken should be `golint` not `golin`